pyccel/parser/syntactic.py

```diff
@@ -1102,9 +1104,11 @@ def _visit_Call(self, stmt):
         elif isinstance(func, DottedName):
             func_attr = FunctionCall(func.name[-1], args)
             func = DottedName(*func.name[:-1], func_attr)
+        elif isinstance(func,IndexedElement):
+            func = IndexedFunctionCall(func.base, args, func.indices)
+            # func = KernelCall(func.base, args, 1, 1, func.indices)
```

Comment:
Can this comment be deleted?

Reply:
fixed [9e97280](https://github.com/pyccel/pyccel-cuda/pull/42/commits/9e97280030ace0736e1356d4be3a72d6f80156bf)